conf/switches.conf.defaults

```diff
@@ -29,7 +29,7 @@ VlanMap = Y
 RoleMap = N
 ExternalPortalEnforcement = N
 
-mode = testing
+mode = production
```

Comment:
Does this really belong in this PR ?

From what I know this is already in devel

Reply:
Fixed